gradle/libs.versions.toml

```diff
@@ -1,18 +1,18 @@
 [versions]
-activitycompose = "1.7.2"
-agp = "8.1.1"
+activitycompose = "1.8.1"
+agp = "8.1.4"
 androidxtest = "1.5.0"
-compose-bom = "2023.09.01"
-composecompiler = "1.5.3"
+compose-bom = "2023.10.01"
```

Comment:
Do we understand how kotlin-bom and compose-bom interact with each other? Does compose-bom include the kotlin-bom?

Reply:
ok